src/operator/numpy/np_diagflat_op-inl.h

```diff
@@ -0,0 +1,201 @@
+/*
```

Comment:
Better put this op in `np_init_op` files.

Reply:
Seems like there's no `np_init_op-inl.h`, you could probably rename this `-inl.h` to that.

Reply:
done